packages/cli/src/cmds/capability.ts

```diff
@@ -88,218 +100,53 @@ abstract class CapabilityAddBase extends YargsCommand {
   }
 }
 
-export class CapabilityAddTab extends YargsCommand {
+export class CapabilityAddTab extends CapabilityAddBase {
   public readonly commandHead = `tab`;
   public readonly command = `${this.commandHead}`;
   public readonly description = "Add a tab.";
+  public readonly yargsHelp = "addCapability-Tab";
 
-  public builder(yargs: Argv): Argv<any> {
-    this.params = HelpParamGenerator.getYargsParamForHelp("addCapability-Tab");
-    return yargs.options(this.params);
-  }
-
-  public override modifyArguments(args: { [argName: string]: any }) {
-    CLIUIInstance.updatePresetAnswer("capabilities", args["capabilities"]);
-    delete args["capabilities"];
-    return args;
+  public override generateExculdeCaps(settings: ProjectSettings | undefined): {
+    frontend: boolean;
+    backend: boolean;
+    bot: boolean;
+  } {
+    return { frontend: ProjectSettingsHelper.includeFrontend(settings), backend: true, bot: true };
```

Comment:
parameters are not used here?

Reply:
I see it is `exclude` in the `automaticNpmInstallHandler` signature.